install_ffmpeg.sh

```diff
@@ -1,230 +1,3 @@
 #!/usr/bin/env bash
-
-set -exuo pipefail
```

Comment:
It seems like there were conflicts in this file. Did we have any changes on the `ai-video` branch and if so, it it fine to drop them completely in place of the lpms script?

Reply:
nevermind. the commit hash for the script changed later so I understood what happened